nativelib/src/main/scala/scala/scalanative/posix/utime.scala

```diff
@@ -0,0 +1,15 @@
+package scala.scalanative
+package posix
+
+import scala.scalanative.native._
+
+@extern
+object utime {
+  // TODO: Where should we put those recurrent types?
```

Comment:
We put type aliases in the object that corresponds to header where they are originally defined in C.